README.txt

See the manual.sxw included in this extension, or check the manual online:
http://typo3.org/documentation/document-library/extension-manuals/mail2news/


ChangeLog
12-06-29 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 2.0.3 stable
	* Fix issue #34508: categories are now assigned correctly again from importer records
	* Fixed table misspelling (tx_t3blog_post_cat should be tx_t3blog_cat)
	* Checked compatibiltiy with TYPO3 4.7, tt_news 3.2 and t3blog 1.2
	* Minor update manual

12-06-26 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 2.0.2 stable
	* Fix issue 30860: commas in filenames

11-12-30 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 2.0.1 stable
	* Fix issue 32865: Category names are not recognized
	* Fix issue 32875: Compatibility with TYPO3 v4.6.x (also tested on 4.7-dev)

11-10-05 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 2.0.0 beta
	* Major new features: Now also import email messages into T3BLOG!
	* Very flexible fieldMarkers feature for setting values in any custom database field (see manual)
	* support for adding multiple categories (tt_news or t3blog) to an imported item
	* transformation of readable date fields to tstamps

11-07-16 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.9 stable
	* mail2news is now compatible with TYPO3 v4.5.x (ClearCache method was broken due to changes in TYPO3 core)
	* Improved category recognition & input sanitation
	* Updated manual
	* This will be the last version of this 1.x.x branch, a new version will be released later which may require some configuration changes!

10-09-09 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.8 beta
	* 8054/8806: Fallback to emConf not working when there are no mail2news configuration records

10-05-18 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.7 beta
	* Now supports multiple configurations using tx_mail2news_importer records (allowing use in multi-site installations, etc.)
	* Added scheduler support
	* Reference index is now properly updated after database inserts
	* Now logs to sys_log table
	* Improved error handling (write to sys_log and continue)
	* Optional DB inserts using TCEmain (includes full permissions, history, etc.)
	* Added automatic link recognition in plain text body for urls and email addresses

10-05-06 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.6 beta
	* Now uses API methods for storing images and attachments
	* No more random number prepended to filenames, if file exists, it gets
	prepended with _1, _2, etc. (issue 7678)
	* Fixed bug 7680, dealing with special chars in filenames. Filenames
	have to be converted t3lib_cs::specCharsToASCII for ImageMagick to
	handle them
	* Fixed bug 3783, better formatting/improved html markup of bodytext (now
	preserves br-tags for linebreaks, inserts p-tags at empty lines and removes
	whitespace on empty lines (spc, &nbsp;, or tabs)

10-04-29 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Fixed bug characterset conversion on a UTF-8 TYPO3 setup:
	http://forge.typo3.org/issues/show/6678
	* Applied source formatting and improved code comments using extdeveval

10-04-23 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Added file ext_localconf.php with automatic cli_key registration
	
09-03-06 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.4 beta
	* Added MIME-header decoding and character set conversion for
	image and attachment filenames

09-02-25 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.3 beta
	* Added option to choose news category and subheader/short in messages
	* News category setting in EM has become 'default category' now
	* Category input can be both category title or id
	* Nicer log file output

09-02-20 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.2 beta
	* Cron script now uses TYPO3's cli_dispatch script to execute the cronjob in
	the context of TYPO3
	* Fixed characterset conversion issue with subject-line

09-02-16 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Version 1.9.1 alpha
	* Completely rewritten, now object oriented code, better structured,
	maintainable and extendible
	* Added features:
	* Now also handles images other than jpg
	* Attached files other than images are saved to tt_news field NEWS_FILES
	* Separate max file size settings for images and attachments 
	* Improved processing of multi-part messages, including nested multi-part
	* Added character set conversion with separate character set recognition for
	header and each part of multi-part message
	* Optionally glues/concatenates all text parts of multipart message together
	* Extended mail server configuration, POP3/IMAP, SSL option and port setting

08-10-06

	* Version 1.1.0 beta
	* Added clearCacheCmd for clearing cache of specified pages after new imports
	* Added extension icon

08-10-01 Loek Hilgersom  <typo3extensions@netcoop.nl>

	* Initial release v1.0.0 beta
	* Support multiple images per message
	* Blocks unallowed script access through HTTP (only shell/cron access allowed)


ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext "mail2news".
#
# Auto generated 29-06-2012 11:51
#
# Manual updates:
# Only the data in the array - everything else is removed by next
# writing. "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'Mail to tt_news',
	'description' => 'Import messages from an email account into tt_news or t3blog records. The easiest way to publish content on your site, start "moblogging" with TYPO3!',
	'category' => 'services',
	'author' => 'Loek Hilgersom',
	'author_email' => 'typo3extensions@netcoop.nl',
	'shy' => '',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'module' => '',
	'state' => 'stable',
	'internal' => '',
	'uploadfolder' => 0,
	'createDirs' => '',
	'modify_tables' => '',
	'clearCacheOnLoad' => 0,
	'lockType' => '',
	'author_company' => 'NetCoop.nl',
	'version' => '2.0.3',
	'constraints' => array(
		'depends' => array(
			'php' => '5.2.0-5.3.99',
			'typo3' => '4.1.0-4.7.99',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:27:{s:9:"ChangeLog";s:4:"140e";s:10:"README.txt";s:4:"170f";s:30:"class.tx_mail2news_getmail.php";s:4:"7e6a";s:27:"class.tx_mail2news_imap.php";s:4:"d5e8";s:38:"class.tx_mail2news_scheduler_start.php";s:4:"4fcf";s:29:"class.tx_mail2news_t3blog.php";s:4:"542c";s:29:"class.tx_mail2news_ttnews.php";s:4:"9bb6";s:16:"ext_autoload.php";s:4:"690f";s:21:"ext_conf_template.txt";s:4:"72d2";s:12:"ext_icon.gif";s:4:"d436";s:17:"ext_localconf.php";s:4:"7eec";s:15:"ext_php_api.dat";s:4:"095e";s:14:"ext_tables.php";s:4:"3a9b";s:14:"ext_tables.sql";s:4:"7594";s:15:"getmail_cli.php";s:4:"d563";s:30:"icon_tx_mail2news_importer.gif";s:4:"d436";s:8:"init.php";s:4:"69ad";s:26:"locallang_csh_importer.xml";s:4:"92b3";s:16:"locallang_db.xml";s:4:"2b02";s:7:"tca.php";s:4:"a384";s:45:"Tests/Unit/class.tx_mail2news_getmailTest.php";s:4:"0c12";s:44:"Tests/Unit/class.tx_mail2news_ttnewsTest.php";s:4:"17e4";s:57:"Tests/Unit/Fixtures/class.tx_mail2news_phpunittypo3db.php";s:4:"ea61";s:14:"doc/manual.sxw";s:4:"b05d";s:21:"doc/screenshot-em.png";s:4:"887e";s:19:"doc/wizard_form.dat";s:4:"b8c3";s:20:"doc/wizard_form.html";s:4:"c6a8";}',
	'suggests' => array(
	),
);

?>